components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/device/constants/Constants.java

```diff
@@ -51,11 +51,19 @@
     public static final String SEPARATED_WITH_SPACE = " ";
     public static final String ERROR = "error";
     public static final String ERROR_DESCRIPTION = "error_description";
+    public static final String DEVICE_DO_ENDPOINT = "/authenticationendpoint/device.do";
+    public static final String DEVICE_SUCCESS_DO_ENDPOINT = "/authenticationendpoint/device_success.do";
+    public static final String USRCDE_QNTFR_CONSTRAINT = "USRCDE_QNTFR_CONSTRAINT";
+
+    public static final String EXPIRY_TIME_PATH = "OAuth.SupportedGrantTypes.SupportedGrantType.ExpiryTime";
+    public static final String CONF_KEY_SET = "OAuth.SupportedGrantTypes.SupportedGrantType.KeySet";
+    public static final String CONF_USER_CODE_LENGTH = "OAuth.SupportedGrantTypes.SupportedGrantType.UserCodeLength";
+    public static final String RESPNSE_TO_STRING_PATH = "OAuth.SupportedGrantTypes.SupportedGrantType.ResponseToString";
 
     // Configurable values.
     public static final int KEY_LENGTH = 6;
-    public static final long EXPIRES_IN_VALUE = 600000L;
-    public static final int INTERVAL_VALUE = 5000;
+    public static final long EXPIRES_IN_VALUE = 600L;  //in seconds
```

Comment:
EXPIRES_IN_VALUE better to change this to EXPIRES_IN_MILLISECONDS
INTERVAL_VALUE better to change this to  INTERVAL_MILLISECONDS

and both needs to be in same time resolution. (milliseconds)